src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelJsonProtocol.scala

```diff
@@ -9,45 +9,86 @@ import spray.json.DefaultJsonProtocol._
 import spray.routing.{MalformedRequestContentRejection, RejectionHandler}
 import spray.routing.directives.RouteDirectives.complete
 
-object ModelJsonProtocol {
+//Mix in one of these with your AttributeFormat so you can serialize lists
+//This also needs mixing in with an AttributeFormat because they're symbiotic
+sealed trait AttributeListSerializer {
+  def writeListType(obj: Attribute): JsValue
+  def readListType(json: JsValue): Attribute
+
+  def writeAttribute(obj: Attribute): JsValue
+  def readAttribute(json: JsValue): Attribute
+}
 
-  implicit val impMethod = jsonFormat8(MethodRepository.Method)
-  implicit val impConfiguration = jsonFormat9(MethodRepository.Configuration)
+//Serializes attribute lists to e.g. [1,2,3]
+//When reading in JSON, assumes that [] is an empty value list, not an empty ref list
+trait PlainArrayAttributeListSerializer extends AttributeListSerializer {
```

Comment:
`PlainArrayAttributeListSerializer` is never used as far as I can see. Is this the one orchestration would need to talk to cromwell? Maybe we only pull this in once we realize we need it?